exporter/opentelemetry-exporter-zipkin-json/src/opentelemetry/exporter/zipkin/json/__init__.py

```diff
@@ -96,7 +97,7 @@
 class ZipkinExporter(SpanExporter):
     def __init__(
         self,
-        protocol: Protocol,
+        version: Protocol = Protocol.V2,
```

Comment:
I think there is not default in specification. Should we make V2 json as default for this release?

Reply:
the old default was v2 protobuf, i think v2 makes sense here.

Reply:
I had sent a PR to make v2 default but it was rejected and spec was updated with this instead. 
> Addtionally, the following environment variables are reserved for future usage in Zipkin Exporter configuration:
>
> - `OTEL_EXPORTER_ZIPKIN_PROTOCOL`
>
> This will be used to specify whether or not the exporter uses v1 or v2, json, thrift or protobuf. As of 1.0 of the specification, there is no specified default, or configuration via environment variables.

Reply:
Ah interesting. I guess we make a choice for the default used for the proto package since only v2 is implemented in that protocol now. I'm ok w/ using the same default for json